assets/css/default.css

```diff
@@ -1,3 +1,43 @@
+.channel-owner {
+  background-color: #008BEC;
+  color: beige;
+  border-radius: 9px;
+  padding: 1px 6px;
+}
+
```

Comment:
Hovering over the channel owner makes it difficult to see.

![image](https://user-images.githubusercontent.com/14208607/52165065-4b935080-26c1-11e9-8abb-6f89406c3d80.png)

I would consider adding a `:hover` with the colors used by the subscribe button, something that looks something like:
```css
.channel-owner:hover {
  background-color: #a0a0a0;
  color: rgba(35, 35, 35, 1);
}
```